source/javascripts/controllers/_GenericFileController.js.erb

```diff
@@ -22,20 +24,29 @@ angular.module("BitriseWorkflowEditor").controller("GenericFileController", func
 	viewModel.genericFileUploader;
 	viewModel.genericFileSettings = {};
 
-	function loadGenericFiles() {
-		viewModel.loadGenericFilesProgress.start("<%= data[:strings][:code_signing][:generic_file][:load_progress][:in_progress] %>");
+	viewModel.init = function() {
+		loadGenericFiles();
+	}
+
+	function loadGenericFiles(shouldClearPreviousLoad) {
+		if (shouldClearPreviousLoad === undefined) {
+			shouldClearPreviousLoad = false;
```

Comment:
same, I don't think we need this if we only check it as a boolean

Reply:
Yep, it's just to make it more explicit what will be the default value, but we can remove it, functionally it's okay without this too.